src/components/activity-page/managed-interactive/managed-interactive.tsx

```diff
@@ -388,6 +389,18 @@ export const ManagedInteractive: React.ForwardRefExoticComponent<IProps> = forwa
   const hasBorder = isManagedInteractive || (isInteractive && !hideQuestionHeader);
   const className = classNames("runtime-container", {"has-question-number": hasQuestionNumber, "has-border": hasBorder});
 
+  useEffect(() => {
+    async function watchFeedback() {
+      const answer = await getAnswerMetadata(interactiveId);
+      if (answer?.id) {
+        return watchQuestionLevelFeedback(answer.id, (fb) => {
+          setFeedback(fb);
+        });
+      }
+    }
+    watchFeedback();
+  }, [interactiveId]);
```

Comment:
Same question here - can we return a function that "unwatches" the feedback when it unmounts?